spec/function.dd

```diff
@@ -1219,39 +1219,57 @@ $(H2 $(LNAME2 parameters, Function Parameters))
+        while $(D count) will be read and written to, and $(D errno) will be set to a value from
+        within the function.
+        This approach gives a semantic meaning to the parameters and allow the compiler to potentially
+        optimize the generated code.)
 
-        $(UL
-        $(LI The function declaration makes it clear what the inputs and
-        outputs to the function are.)
-        $(LI It eliminates the need for IDL (interface description language) as a separate language.)
-        $(LI It provides more information to the compiler, enabling more
-        error checking and
-        possibly better code generation.)
+        $(P When talking about function parameter, the $(I parameter) is what the function prototype defines,
+        and the $(I argument) is the value that will $(I bind) to it.
+        For example, using the previous definition of $(D read):
         )
+------
+void main()
+{
+    size_t a = 42;
+    int b;
+    int r = read("Hello World", a, b);
+}
+------
+       $(P In this example, the argument $(D "Hello World") gets bind to $(D input),
+       $(D a) gets bind to $(D count) and $(D b) to $(D errno).
+       )
 
     $(TABLE_2COLS Parameter Storage Classes,
     $(THEAD Storage Class, Description)
-    $(TROW $(I none), parameter becomes a mutable copy of its argument)
-
-    $(TROW $(D in), defined as $(D scope const).  However $(D in) has not yet been properly
-    implemented so it's current implementation is equivalent to $(D const).  It is recommended
-    to avoid using $(D in) until it is properly defined and implemented.  Use $(D scope const)
-    or $(D const) explicitly instead.)
-    $(TROW $(D ref), parameter is passed by reference)
-    $(TROW $(D out), parameter is passed by reference and initialized upon function entry with the default value
-    for its type)
+    $(TROW $(I none), The parameter will be a mutable copy of its argument)
+    $(TROW $(D in), The parameter is an input to the function. Input parameters behaves as if they have
+    the $(D const scope) storage classes. Input parameters can be passed by reference.
+    If they get passed by reference, unlike with $(D ref), rvalues (such as literals) can be bound to them.
+    Types that would trigger a side effect if passed by value (such as types with postblit,
+    copy constructor, or destructor), and types which cannot be copied, e.g. if their copy constructor is
+    marked as $(D @disable), will always be passed by reference. Dynamic arrays, classes,
+    associative arrays, function pointers and delegates will always be passed by value,
+    to allow for covariance. If a parameter is of a type that does not fall in one of those categories,
+    whether or not it is passed by reference is implementation defined, and the backend is free
+    to choose the method that will best fit the ABI of the platform.
+    $(B Note: This require the $(D -preview=in) switch available in v2.094.0 or higher.))
```

Comment:
Something is really, really wrong here for the website to be rendered list [this](http://dtest.dlang.io/artifact/website-94abd4f028d79da8757bca9d1cf8d5d3e7f68de3-3a34ed1ce175aef09784ac4a45b2da1f/web/spec/function.html#param-storage).

Reply:
Looks like the problem is your use of commas.  There should be a ddoc macro to use instead.

Reply:
Should be `$(COMMA)`.